samples/logging/custom-factory/Core_10/Sample/Program.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using NServiceBus;
+using NServiceBus.Logging;
+
+var host = Host.CreateDefaultBuilder(args)
+    .UseConsoleLifetime()
+    .UseNServiceBus(_ =>
+    {
+        #region ConfigureLogging
+        var loggerDefinition = LogManager.Use<ConsoleLoggerDefinition>();
+        loggerDefinition.Level(LogLevel.Info);
+
+        var endpointConfiguration = new EndpointConfiguration("Samples.Logging.CustomFactory");
+
+        #endregion
```

Comment:
Is there a good reason to include the endpoint configuration in the snippet?

```suggestion
        #region ConfigureLogging
        var loggerDefinition = LogManager.Use<ConsoleLoggerDefinition>();
        loggerDefinition.Level(LogLevel.Info);
        #endregion
        var endpointConfiguration = new EndpointConfiguration("Samples.Logging.CustomFactory");
```